### PR TITLE
Create user account static pages

### PR DESCRIPTION
**Pull request by faun**
When trying to create an account where the email address is already taken, redirect to a page that contains more information.

See also:
- https://github.com/percy/percy-api/pull/943

### CODE REVIEW

File: tests/acceptance/password-reset.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  setupAcceptance({authenticate: false});
+
+  setupSession(function(server) {
+    this.loginUser = false;
+    this.server = server;
+  });
+
+  it('shows page with password reset message ', async function() {
+    await visit('/auth/password-reset');
+    await percySnapshot(this.test.fullTitle());
```
Comment (djones) [inline tests/acceptance/password-reset.js]: These pages didn't show up in the Percy build. Not sure why...

File: app/templates/duplicate-email.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{fixed-top-header}}
+
+<div class="container-full container-vh bg-white-pattern flex items-center justify-center">
+  {{{status-panel
+    inlineSvgName="email-large-icon"
+    mainText="Uh Oh!"
+    secondaryText="We already have an account for that email addresss.<br/><strong>Please sign in first and then add a <a href='/settings/connected-accounts'>connected account</a> or contact support for assistance.</strong>"
```
Comment (cadeParade) [inline app/templates/duplicate-email.hbs]: Typo here: `sss`

File: app/routes/duplicate-email.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
```
Comment (cadeParade) [inline app/routes/duplicate-email.js]: I don't think you need these route files if there's nothing in them.

File: app/templates/duplicate-email.hbs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{{fixed-top-header}}
+
+<div class="container-full container-vh bg-white-pattern flex items-center justify-center">
+  {{{status-panel
```
Comment (cadeParade) [inline app/templates/duplicate-email.hbs]: Three handlebars here...While that is a thing you can do, I don't think it's what you want here.

File: tests/acceptance/password-reset.js (hunk 1 above)
Comment (faun) [reply tests/acceptance/password-reset.js]: Fixed by adding `-test` suffix to the file name.

**Pull request merged**